requirements/common.txt

```diff
@@ -169,4 +169,4 @@ beautifulsoup4==4.6.0
 # We integrate with these tightly, so often it makes sense to pin a
 # version from Git rather than a release.
 -e "git+https://github.com/zulip/python-zulip-api.git@0.3.2#egg=zulip==0.3.2&subdirectory=zulip"
--e "git+https://github.com/zulip/python-zulip-api.git@0.3.2#egg=zulip_bots==0.3.2&subdirectory=zulip_bots"
+-e "git+https://github.com/eeshangarg/python-zulip-api.git@automated-markdown-docs#egg=zulip_bots==0.3.3&subdirectory=zulip_bots"
```

Comment:
you should increase PROVISION_VERSION if this package requires a newer version of a dependency to work.